test/test_jit.py

```diff
@@ -11950,10 +11954,10 @@ def test_milstm_cuda(self):
         self.assertGraphContainsExactly(
             forward_graph, 'prim::FusionGroup', 1, consider_subgraphs=True)
         self.assertExpectedGraph(forward_graph, subname='forward')
-
+        FileCheck().check("DifferentiableGraph").check_next("TupleConstruct") \
+            .check_next("return").check("FusionGroup").run(str(forward_graph))
```

Comment:
It would be nice if we could keep around the milstm forward/backward expect files as documentation. It's not as important as the lstm forward/backward expect files though